### PR TITLE
3.13 Imports formatting

### PR DESCRIPTION
**Pull request by Krigpl**
Not sure what's the benefit of this repetitiveness.

### CODE REVIEW

File: pyguide.md [hunk 1]
```diff
@@ -2009,9 +2009,11 @@ import tensorflow as tf
 from book.genres import scifi
 from myproject.backend.hgwells import time_machine
 from myproject.backend.state_machine import main_loop
-from otherproject.ai import body
-from otherproject.ai import mind
-from otherproject.ai import soul
+from otherproject.ai import (
```
Comment (maaktweluit) [inline pyguide.md:2012]: It looks like this fits on a 80 character line, agree with this rule once the line is 80+

```suggestion
from otherproject.ai import body, mind, soul
```

**Review by maaktweluit — COMMENTED** [review]
better then it was, but have one more suggestion

**Review by Wiezzel — APPROVED** [review]

File: pyguide.md (hunk 1 above)
Comment (Wiezzel) [reply pyguide.md:2012]: I'm kinda torn between here. On one hand multiple imports per line are more compact and look better. On the other, it's easier to track modifications in code with one import per line.

Comment (maaktweluit) [reply pyguide.md:2012]: now i'm torn too :P

One line one thought > compact code

so maybe leave as igor proposed?

Comment (Krigpl) [reply pyguide.md:2012]: I'm fine either way

**Review by mfranciszkiewicz — APPROVED** [review]
This greatly improves readability, I don't personally care about the line count

**Review by maaktweluit — APPROVED** [review]
Ignore my comment, readability over all!

**Pull request merged**